.github/workflows/wait-for-akd-publish.sh

```diff
@@ -0,0 +1,46 @@
+# Finds expected version of a crate in another crate's Cargo.toml file
```

Comment:
You should have a shell identifier here or we're going to run into all sorts of problems if we change the CI pipeline ever.

Something like:
```
#!/bin/bash
```
if you expect bash